### PR TITLE
[Coordinated, template] Use CloudEvent aware modules.

### PR DESCRIPTION
**Pull request by ericbottard**
This PR uses temporary coordinates, showcases where edits will be needed.

### CODE REVIEW

File: config/riff-streaming.yaml [hunk 1]
```diff
@@ -6567,7 +6567,7 @@ subjects:
 ---
 apiVersion: v1
 data:
-  gatewayImage: bsideup/liiklus:0.9.0
+  gatewayImage: ctr.run/github.com/bsideup/liiklus:00385b688bc34d6fdbe2b5eeb39daa26e09f8b02
```
Comment (ericbottard) [inline config/riff-streaming.yaml]: Or wait for https://github.com/bsideup/liiklus/pull/252 to be merged and use a released version

File: config/riff-streaming.yaml [hunk 2]
```diff
@@ -6589,7 +6589,7 @@ metadata:
 ---
 apiVersion: v1
 data:
-  processorImage: projectriff/streaming-processor-native:1.0.0-SNAPSHOT-20191223112938-672714499ed0c511@sha256:41ad7b6b350370b5e53cc718eb7cde49d7f9f7e22e33a3204ae0a4158f798819
+  processorImage: ericbottard/processor-native@sha256:9e1a16a68bd58e61c34e6bfc0af4c25fbd3f04fa2a045f0cd156c0ed71c8a641
```
Comment (ericbottard) [inline config/riff-streaming.yaml]: Or wait for https://github.com/projectriff/streaming-processor/pull/42 to be merged

**Comment by scothis** [comment]
@ericbottard is this in scope for 0.5?

**Comment by ericbottard** [comment]
> @ericbottard is this in scope for 0.5?

I would say no, although not really for me to decide. Also depends on the state of the liiklus PR. I would rather have us depend on a released version of liiklus for our releases.

cc @bsideup @jldec @markfisher

**Comment by bsideup** [comment]
@ericbottard not sure when 0.5 is planned but I am hoping to merge the CloudEvents PR this week. It is nearly done implementation wise, just need to finish the extensions (have it working but a bit concerned about the APIs used)

File: config/riff-streaming.yaml (hunk 1 above)
Comment (fbiville) [reply config/riff-streaming.yaml]: you can now use `bsideup/liiklus:0.10.0-rc1`

Comment (ericbottard) [reply config/riff-streaming.yaml]: Let's kill two birds with one stone once the streaming-processor PR is merged as well...

**Comment by sbawaska** [comment]
superceded by #333

**Pull request closed without merging**